contracts/CENNZnetBridge.sol

```diff
@@ -102,9 +102,10 @@ contract CENNZnetBridge is Ownable {
         validators[proof.validatorSetId] = newValidators;
         activeValidatorSetId = proof.validatorSetId;
 
-        // return any accumlated fees to the sender as a reward
+        // return any accumulated fees to the sender as a reward
         uint reward = address(this).balance;
-        payable(msg.sender).transfer(reward);
+        (bool sent, ) = msg.sender.call{value: reward}("");
```

Comment:
wondering if this will succeed when `reward` is `0`? (we would want it to)

Reply:
Tested this.. on ropsten
https://ropsten.etherscan.io/tx/0x71891e2a496aee6f4f6d00092222b7802711ddfc6540bc8bf9ac7a37de3ed3b9